src/checklist/CheckItems/CustomCheckItems.tsx

```diff
@@ -0,0 +1,253 @@
+import { Button, Checkbox, Scrim, TextField } from '@equinor/eds-core-react';
+import React, { useEffect, useState } from 'react';
+import styled from 'styled-components';
+import EdsIcon from '../../components/icons/EdsIcon';
+import { AsyncStatus, CustomCheckItem } from '../../services/apiTypes';
+import { ProcosysApiService } from '../../services/procosysApi';
+import { COLORS, SHADOW } from '../../style/GlobalStyles';
+import updateCustomCheck from '../../utils/updateCustomCheck';
+import CheckHeader from './CheckHeader';
+
+const CustomCheckItemWrapper = styled.div`
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-top: 8px;
+    & p {
+        margin: 0;
+    }
+`;
+
+const NewCustomItemWrapper = styled.div`
+    display: flex;
+    margin-bottom: 16px;
+    margin-top: 16px;
+    & > button,
+    button:disabled {
+        margin-left: 24px;
+        margin-right: 12px;
+    }
+`;
+
+const RightWrapper = styled.div`
+    display: flex;
+    align-items: center;
+    & button {
+        margin-right: 5px;
+        margin-left: 1px;
+    }
+`;
+
+const DeletionPopup = styled.div`
+    border-radius: 5px;
+    background-color: ${COLORS.white};
+    padding: 15px;
+    box-shadow: ${SHADOW};
+    & > :last-child {
+        margin-left: 15px;
+    }
+`;
+
+type CustomCheckItemsProps = {
```

Comment:
I find it easier to read when the prop type for the component is directly above the component, and the other types are between the styling types and the prop type. You might not agree.